src/main/java/org/phyloref/jphyloref/JPhyloRef.java

```diff
@@ -51,7 +56,7 @@ public int execute(String[] args) {
     try {
       cmdLine = new DefaultParser().parse(opts, args);
     } catch (ParseException ex) {
-      System.err.println("Could not parse command line options: " + ex);
+      logger.error("Could not parse command line options: {}", (Object) ex);
```

Comment:
Are you sure you need the typecast to `Object` here?

Reply:
I replaced it with `ex.toString()` in abc0a43 to make it more readable.